reesegreen2014/TripTrove
# Feature/book trip

#### What does this PR do?
This PR defines the structure, styling, and DOM updates for the book-trip feature of the application. 
#### Where should the reviewer start?
Please start by pulling down local changes using `git pull origin feature/book-trip` and running `npm start` to start the application. Ensure you are also running the back-end API. 
#### How should this be manually tested?
Login to the application by clicking the "Login" button in the top right hand corner of the application. NOTE: All users share the same pattern of credentials. Username is travelerID where ID represents the userID. Password is travel. Login and observe the DOM Updates. Click "Book A Trip" to test the auto-scroll styling. Test multiple different booking routes, including testing error handling (by not entering anything in one of the fields and attempting to book).
#### Any background context you want to provide?
#### What are the relevant tickets (if any)?
Relevant tickets will be closed upon completion of this merge.
#### Screenshots (if appropriate)
![image](https://github.com/reesegreen2014/TripTrove/assets/155911680/87613fc6-6b85-41cd-a633-d18cbc3f2e58)
![image](https://github.com/reesegreen2014/TripTrove/assets/155911680/0c8103bd-bd72-4aa8-8e8a-8ca75bb9c1f8)
![image](https://github.com/reesegreen2014/TripTrove/assets/155911680/4fc2f4a0-42a0-4396-8d0e-336e6d5b87e1)

#### Questions:
